src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -141,7 +141,7 @@ export interface SavedObjectsServiceSetup {
    * }
    * ```
    */
-  registerType: (type: SavedObjectsType) => void;
+  registerType: <Attributes = any>(type: SavedObjectsType<Attributes>) => void;
```

Comment:
Added generic for the attributes on `SavedObjectsType`, `SavedObjectsTypeManagementDefinition` and underlying types. However for BWC, I was forced to default to `any` instead of `unknown` for the public API.

Reply:
How much does this break if we switch to `unknown`? I'm guessing a lot 😓

Reply:
Yea, basically all types registering a `getUrl` or `getTitle` management handler, which is a lot. This is why I don't think we should do it in this PR. I can open a follow-up and ping the teams, though